scripts/fetch

```diff
@@ -9,22 +9,22 @@ cores=$(nproc)
 manifest_dir="${MANIFEST_DIR?}"
 manifest_repo="${MANIFEST_REPO?}"
 
-mkdir -p "${base_dir}" "${manifest_dir}"
+mkdir --parents "${base_dir}" "${manifest_dir}"
 
-rsync -Pav "${manifest_repo}/" "${manifest_dir}/"
+rsync -P --archive --verbose "${manifest_repo}/" "${manifest_dir}/"
```

Comment:
similarly... I `rsync -av` is a common 'phrase'. I would have to look up the man page to check what `--archive` and `--verbose` do.

Reply:
Here I would argue that anyone who was read the `rsync` manpage knows that `-a` and `--archive` are equivalent. But someone who has not read it is probably better of with `--archive`. Same with `--verbose` which is more explicit than `-v`. You would really need to check what `--verbose` does compared to `-v` ;-) ?

Reply:
> You would really need to check what `--verbose` does compared to `-v`

Yes! I didn't actually know `-v` was for verbose in rsync; for some reason its just habit for me to type it.

Reply:
Again I feel we should optimize for people new to this stuff, than those of us who have been memorizing shell incantations for 20 years. Verbosity is good for security. Make people easily see options and think about what they do as a standard practice.